tests/test_config.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of the github2jira project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2021 Red Hat, Inc.
+#
+
+import sys
+import os
+
+import mock
+import pytest
+
+sys.path.insert(1, sys.path[0] + "/..")
+import githublib  # noqa: E402
+
+
+def mockenv(**envvars):
+    return mock.patch.dict(os.environ, envvars)
```

Comment:
This is very odd. I think you can decorate using `mock.patch` directly, I do not understand why this wrapper is needed.

Reply:
it is a wrapper which i find cleaner
https://newbedev.com/python-mock-patch-os-environ-and-return-value

Reply:
It is not a standard way to do things and it is very confusing to a new reader.
But I do not want to argue about this, I leave it up to you.